README.md

```diff
@@ -2096,6 +2096,37 @@ ANSWERS.addComponent('SearchBar', {
 })
 ```
 
+The SDK also offers an `ANSWERS.registerTemplate` function. This is to register
+a template string to any given template name in our template bundle.
+
+```js
+  /**
+   * Compile and add a template
+   * @param {string} templateName The unique name for the template
+   * @param {string} template The handlebars template string
+   */
+  registerTemplate (templateName, template)
+```
+
+If the template name does not exist, a new template name is created. If
+it does exist, it overrides the current template for the template name. This
+allows you override default template names.
```

Comment:
nit: What do you think about describing these as entries in a registry/dictionary keyed off of template name? I think it's already currently implied, but unsure if we should explicitly say it